### PR TITLE
[release-4.6] Bug 1939059: sap license management logs gatherer

### PR DESCRIPTION
**Pull request by Sergey1011010**
Backport

```
The SAP installation may fail in license management pod, so We need error logs to detect the failure.

Get the filtered error logs as followings.

# oc logs deploy/license-management-l4rvh
Found 2 pods, using pod/license-management-l4rvh-74595f8c9b-flgz9
+ iptables -D PREROUTING -t nat -j VSYSTEM-AGENT-PREROUTING
+ true
+ iptables -F VSYSTEM-AGENT-PREROUTING -t nat
+ true
+ iptables -X VSYSTEM-AGENT-PREROUTING -t nat
+ true
+ iptables -N VSYSTEM-AGENT-PREROUTING -t nat
iptables v1.6.2: can't initialize iptables table `nat': Permission denied
Perhaps iptables or your kernel needs to be upgraded.
```
Tested on SAP cluster with different substrings.

## Categories

- [ ] Bugfix
- [X] Enhancement
- [ ] Backporting
- [ ] Others (CI, Infrastructure, Documentation)

## Sample archive

No changes

## Documentation

- `docs/gathered-data.md`

## Unit Tests

Not backporting `gater_logs_test.go` because we would need to update k8s version for them to work. 

## Privacy

Yes. There are no sensitive data in the newly collected information.

## References

https://issues.redhat.com/browse/CCXDEV-4172
https://bugzilla.redhat.com/show_bug.cgi?id=1939059
https://access.redhat.com/articles/5100521#license-manager-cannot-be-initialized

### CODE REVIEW

**Comment by Sergey1011010** [comment]
@tremes in order to bring logs gathering tests I needed to update k8s api version. WDYT? Do we keep it here?

File: pkg/gather/clusterconfig/sap_config.go [hunk 1]
```diff
@@ -53,8 +50,7 @@ func gatherSAPConfig(ctx context.Context, dynamicClient dynamic.Interface, coreC
 	sccToGather := []string{"anyuid", "privileged"}
 	crbToGather := []string{"system:openshift:scc:anyuid", "system:openshift:scc:privileged"}
 
-	datahubsList, err := dynamicClient.Resource(datahubGVR).List(ctx, metav1.ListOptions{})
-
+	datahubsList, err := dynamicClient.Resource(datahubGroupVersionResource).List(ctx, metav1.ListOptions{})
```
Comment (tremes) [inline pkg/gather/clusterconfig/sap_config.go:53]: Can you please do the similar change also for `pkg/gather/clusterconfig/sap_pods.go`? Thanks

Comment (Sergey1011010) [reply pkg/gather/clusterconfig/sap_config.go:53]: done

**Comment by tremes** [comment]
I went through the code and did locally a basic "smoke" test run. 
/lgtm

**Pull request merged**